Java 17でテストが落ちる

```
Error:  Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 1.69 s <<< FAILURE! - in io.jenkins.plugins.codeInsights.framework.FileTransferServiceImplTest
Error:  io.jenkins.plugins.codeInsights.framework.FileTransferServiceImplTest.copyFromWorkspaceToLocal copies target file  Time elapsed: 1.676 s  <<< ERROR!
java.lang.ExceptionInInitializerError
	at com.thoughtworks.xstream.XStream.setupConverters(XStream.java:811)
	at com.thoughtworks.xstream.XStream.<init>(XStream.java:574)
	at com.thoughtworks.xstream.XStream.<init>(XStream.java:496)
	at com.thoughtworks.xstream.XStream.<init>(XStream.java:465)
	at com.thoughtworks.xstream.XStream.<init>(XStream.java:411)
	at com.thoughtworks.xstream.XStream.<init>(XStream.java:378)
	at hudson.util.XStream2.<init>(XStream2.java:113)
	at hudson.model.Run.<clinit>(Run.java:[250](https://github.com/T45K/Bitbucket-Server-Code-Insights-plugin/runs/5692554909?check_suite_focus=true#step:4:250)5)
	at org.objenesis.instantiator.sun.SunReflectionFactoryInstantiator.newInstance(SunReflectionFactoryInstantiator.java:48)
	at org.objenesis.ObjenesisBase.newInstance(ObjenesisBase.java:73)
	at org.objenesis.ObjenesisHelper.newInstance(ObjenesisHelper.java:44)
	at org.spockframework.mock.runtime.MockInstantiator$ObjenesisInstantiator.instantiate(MockInstantiator.java:45)
	at org.spockframework.mock.runtime.MockInstantiator.instantiate(MockInstantiator.java:31)
	at org.spockframework.mock.runtime.ByteBuddyMockFactory.createMock(ByteBuddyMockFactory.java:73)
	at org.spockframework.mock.runtime.ProxyBasedMockFactory.create(ProxyBasedMockFactory.java:42)
	at org.spockframework.mock.runtime.JavaMockFactory.createInternal(JavaMockFactory.java:58)
	at org.spockframework.mock.runtime.JavaMockFactory.create(JavaMockFactory.java:38)
	at org.spockframework.mock.runtime.CompositeMockFactory.create(CompositeMockFactory.java:42)
	at org.spockframework.lang.SpecInternals.createMock(SpecInternals.java:47)
	at org.spockframework.lang.SpecInternals.createMockImpl(SpecInternals.java:302)
	at org.spockframework.lang.SpecInternals.createMockImpl(SpecInternals.java:292)
	at org.spockframework.lang.SpecInternals.StubImpl(SpecInternals.java:137)
	at io.jenkins.plugins.codeInsights.framework.FileTransferServiceImplTest.copyFromWorkspaceToLocal copies target file(FileTransferServiceImplTest.groovy:15)
Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make field private final java.util.Comparator java.util.TreeMap.comparator accessible: module java.base does not "opens java.util" to unnamed module @13b6d03
	at com.thoughtworks.xstream.core.util.Fields.locate(Fields.java:39)
	at com.thoughtworks.xstream.converters.collections.TreeMapConverter.<clinit>(TreeMapConverter.java:50)
	... 23 more

Error:  io.jenkins.plugins.codeInsights.framework.FileTransferServiceImplTest.readFile returns of remote file contents  Time elapsed: 0.002 s  <<< ERROR!
java.lang.NoClassDefFoundError: Could not initialize class hudson.model.Run$SpockMock$374471746
	at org.objenesis.instantiator.sun.SunReflectionFactoryInstantiator.newInstance(SunReflectionFactoryInstantiator.java:48)
	at org.objenesis.ObjenesisBase.newInstance(ObjenesisBase.java:73)
	at org.objenesis.ObjenesisHelper.newInstance(ObjenesisHelper.java:44)
	at org.spockframework.mock.runtime.MockInstantiator$ObjenesisInstantiator.instantiate(MockInstantiator.java:45)
	at org.spockframework.mock.runtime.MockInstantiator.instantiate(MockInstantiator.java:31)
	at org.spockframework.mock.runtime.ByteBuddyMockFactory.createMock(ByteBuddyMockFactory.java:73)
	at org.spockframework.mock.runtime.ProxyBasedMockFactory.create(ProxyBasedMockFactory.java:42)
	at org.spockframework.mock.runtime.JavaMockFactory.createInternal(JavaMockFactory.java:58)
	at org.spockframework.mock.runtime.JavaMockFactory.create(JavaMockFactory.java:38)
	at org.spockframework.mock.runtime.CompositeMockFactory.create(CompositeMockFactory.java:42)
	at org.spockframework.lang.SpecInternals.createMock(SpecInternals.java:47)
	at org.spockframework.lang.SpecInternals.createMockImpl(SpecInternals.java:302)
	at org.spockframework.lang.SpecInternals.createMockImpl(SpecInternals.java:292)
	at org.spockframework.lang.SpecInternals.StubImpl(SpecInternals.java:133)
	at io.jenkins.plugins.codeInsights.framework.FileTransferServiceImplTest.readFile returns of remote file contents(FileTransferServiceImplTest.groovy:33)
```

--- a/.github/workflows/execute-test.yaml
+++ b/.github/workflows/execute-test.yaml
@@ -12,7 +12,7 @@ jobs:
     strategy:
       matrix:
         os: [ macos-latest, windows-latest, ubuntu-latest ]
-        java: [ 8, 11 ]
+        java: [ 8, 11 ,17 ]
     steps:
       - uses: actions/checkout@v3
         with:

--- a/pom.xml
+++ b/pom.xml
@@ -14,7 +14,7 @@
     <packaging>hpi</packaging>
     <properties>
         <revision>0.3.0</revision>
-        <jenkins.version>2.235.1</jenkins.version>
+        <jenkins.version>2.340</jenkins.version>
         <java.level>8</java.level>
         <gitHubRepo>T45K/Bitbucket-Server-Code-Insights-plugin</gitHubRepo>
         <maven.compiler.source>1.8</maven.compiler.source>

